camel/models/base_model.py

```diff
@@ -14,11 +14,24 @@
 from abc import ABC, abstractmethod
 from typing import Any, Dict, List
 
+from camel.typing import ModelType
+
 
 class BaseModelBackend(ABC):
     r"""Base class for different model backends.
     May be OpenAI API, a local LLM, a stub for unit tests, etc."""
 
+    def __init__(self, model_type: ModelType,
```

Comment:
Okay, indeed with my implementation the constructor must be there in place in the base class.